stable/datadog/templates/autoconf-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
The YAML delimiters at the top of the configmaps trip up `helm delete` for some reason (and CI fails). You'll need to remove them.